woocommerce/woocommerce-ios
# Shipping Labels: Enable feature flag for adding new packages

Closes: #3909

## Description

This PR enables the feature flag for adding/creating new packages (custom and service packages) when creating a shipping label, to release this feature to the public.

**Question:** In https://github.com/woocommerce/woocommerce-ios/issues/4595 it says that we plan to follow that release order, which means releasing the Add Payment Method feature before this Create Package feature. These features aren't dependent on each other, so it should be ok to release this first. Do we want to wait to follow that release order anyway?

## Changes

Updated `shippingLabelsAddCustomPackages` to true for `DefaultFeatureFlagService`.

## Testing

Setup:

* Make sure that your test store has installed WCShip plugin and payment methods added in the "Shipping Labels" section in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping.
* Remove any packages you have in the "Packaging" section in WPAdmin > WooCommerce > Settings > Shipping > WooCommerce Shipping, so you start testing without any existing packages on your store.

Testing:

1. Open an order detail that is eligible for a shipping label.
2. Tap the button "Create Shipping Label.”
3. Confirm "Ship From" and "Ship To.”
4. In the package details section, tap on “Package Selected.”
5. On the Add New Package screen, create a new custom package (on the Custom Package tab) or select a service package (on the Service Package tab).
6. Tap the Done button and confirm your new package is selected on the "Package Selected" screen.
7. Tap the Done button and confirm your new package is listed in the package details section of the shipping label form.
8. Complete your shipping label purchase.
9. Repeat the steps above with another order. Now that you have existing packages on your store, after step 5 you should see a list of the packages already activated on your store with a "Create new package" button at the bottom of the screen. Tap that button to get to the Add New Package screen. After you add a package, confirm the new package is included (and selected) in the package list.

Scenarios to consider:

* Create a new custom package.
* Activate a service package.
* Add multiple packages in a row.
* Cancel adding a new package at any point in the process.
* Activate all of the available service packages and confirm you see an empty state screen on Add New Package > Service Package tab.

<img src="https://user-images.githubusercontent.com/8658164/132866096-060c34a8-5e2e-400c-b402-82251c3dd74a.gif" width="300px">

## Submitter Checklist

Update release notes:
